### PR TITLE
*: update version and status badge.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: server/util.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ const (
 
 // Version information.
 var (
-	PDReleaseVersion = "0.9.0"
+	PDReleaseVersion = "1.0.0+git"
```
Comment (siddontang) [inline server/util.go]: why +git here?

Comment (disksing) [reply server/util.go]: Make it different from release branches, using the same method as etcd. https://github.com/coreos/etcd/blob/b989e1992fb1186c7a6a5ea9291fae888a3d7971/version/version.go#L29

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
PTAL @nolouch

**Pull request merged**